synapse/app/__init__.py

```diff
@@ -24,7 +24,7 @@
 try:
     python_dependencies.check_requirements()
 except python_dependencies.DependencyException as e:
-    sys.stderr.writelines(e.message)
+    sys.stderr.writelines(e.message)  # noqa: B306
```

Comment:
Should we update this to be `str(e)`?

Reply:
`DependencyException.message` is actually a property that formats it properly, thus the noqa, this is in a few other places too